### DISCUSSION TITLE
Claude 3.5 Sonnet for Copilot cant use

### DISCUSSION THREAD

**Discussion by ziwenze** [community/community#143331] [GitHub Education] [answered]
### Hi everyone,

Copilot policy settings cant see a policy for Anthropic Claude 3.5 Sonnet for Copilot. But I have the student certification benefits for Copilot, and my classmates can use Claude 3.5. I'm quite confused and hope to get some answers.
![image](https://github.com/user-attachments/assets/c4f69e9d-4868-401c-851e-acf4862e81b5)

**Comment by BlurryLight** [comment, marked as answer]
same issue

**Comment by tryptophawa** [comment]
I guess it's because that you need a good internet, which is a good VPN

**Reply by EjuneG** [reply]
Thanks this solves it.
